flink-runtime/src/main/java/org/apache/flink/runtime/rest/messages/job/JobDetailsInfo.java

```diff
@@ -259,6 +261,7 @@ public String getJsonPlan() {
     // ---------------------------------------------------
 
     /** Detailed information about a job vertex. */
+    @Schema(name = "JobDetailsVertexInfo")
     public static final class JobVertexDetailsInfo {
```

Comment:
Why don't we just rename the class? do we need to keep any bw compatibility on this level?

Reply:
We could rename all classes, I just wasn't sure if we want to go down that route purely for documentation work.

Reply:
Compatibility isn't a concern.

Reply:
There are of course advantages to renaming the class, like it being easier to track down the internal representation of something in the API. But I'm not sure if that's worth the hassle of restricting us to unique class names everywhere.